packages/commands/src/index.ts

```diff
@@ -545,6 +545,19 @@ export class CommandRegistry {
       return;
     }
 
+    // Remove modifier key only keystrokes from the current key sequence.
+    // keystrokes that are modifier key(s) plus another key are not affected
+    // intended functionality: Alt then Alt 1 should result to: ['Alt'] then ['Alt 1']
+    // Removing this code results in mod key duplication: ['Alt', 'Alt 1']
+    if (
+      // Check that only a mod key has been pressed
+      (event.altKey && event.key === 'Alt') ||
+      (event.ctrlKey && event.key === 'Ctrl') ||
+      (event.shiftKey && event.key === 'Shift')
+    ) {
```

Comment:
Should we use `CommandRegistry.isModifierKeyPressed(event))` ?
Isn't it supposed to check whether the key pressed is a modifier ?